spec/lib/identity_job_log_subscriber_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe IdentityJobLogSubscriber, type: :job do
+  subject(:subscriber) { IdentityJobLogSubscriber.new }
```

Comment:
this was DRY for the spec, a bunch of the specs call this so it seemed easier to me to define it once